.gitignore

```diff
@@ -2,3 +2,5 @@
 node_modules
```

Comment:
Should `node_modules` have a slash at the end like you did for `build` and `coverage`?
What goes in the `build` directory?